python/correctness/suppressed-exception-handling-finally-break.yaml

```
@@ -34,8 +34,10 @@ rules:
             ...
             return ...
   message: >-
-      having a `break`, `continue`, or `return` in a `finally` block will cause strange behaviors including an execption to be not caught
+      Having a `break`, `continue`, or `return` in a `finally` block will cause strange behaviors, like exceptions not being caught.
   metadata:
+    references:
+    - https://www.python.org/dev/peps/pep-0601/#rejection-note
```

This PEP got rejected, so its suggestions are not implemented. I think it's still a good idea to include it, though, because it explains why this requires a lint check as opposed to being handled within the language itself.

The link I'm suggesting below is the Python language reference that defines this behavior, and which the rejected PEP 601 wanted to amend.
```suggestion
    - https://docs.python.org/3/reference/compound_stmts.html#the-try-statement
    - https://www.python.org/dev/peps/pep-0601/#rejection-note
```

---

Perfect, even better! :D